README.adoc

```diff
@@ -49,6 +49,29 @@ To run on an another port use `--server.port=...`:
 java -jar employee-rostering-distribution/target/employee-rostering-distribution-*.jar --server.port=18080
 ----
 
+== Benchmark Application
+
+After building the project above, run the benchmark application with:
+
+[source,shell]
+----
+java -jar employee-rostering-benchmark/target/employee-rostering-benchmark-*.jar
+----
+
+Note: make sure to run the project application and benchmark application on different ports if they are run
+simultaneously.
+
+After benchmarking is complete, a benchmark report will be generated in the `local/benchmarkReport` directory.
+Open the `index.html` file in your browser to view the benchmark results.
+
+=== Configure the benchmarker
+
+To configure the benchmarker, modify the `employeeRosteringBenchmarkConfig.xml` file found in the
+`employee-rostering-benchmark/src/main/resources` directory.
```

Comment:
Am I correct in assuming that the user also needs to rebuild to get the changes in the JAR?

Reply:
That's correct. I'll add a note in the README.